src/api/app/models/attrib.rb

```diff
@@ -161,6 +162,14 @@ def parse_value(value)
     true
   end
 
+  def validate_attrib_type_for_project_attribute
```

Comment:
@saraycp maybe its enough to just do this validation and not required for to add an additional column to the attributes table for now. As far as I know we might not use attributes to mark the projects for version tracking in the near future. @hellcp-work did I understand correctly that the usage of an attribute for this is was just to have an easy starting point for this feature?

Reply:
Ok. I removed the second commit. Let's leave the validation only so far.